changelog.d/7685.bugfix

```diff
@@ -0,0 +1 @@
+Fix some 3PID management endpoints not being reachable on the `/_matrix/client/r0` route.
```

Comment:
when did it break?

Reply:
1.15, fixed in https://github.com/matrix-org/synapse/pull/7685/commits/3b2ac10d2f906546901cd7aba1c6358758281af8